Test/QuickCheck/Text.hs

```diff
@@ -112,7 +112,7 @@ withNullTerminal action =
   newTerminal (const (return ())) (const (return ())) >>= action
 
 terminalOutput :: Terminal -> IO String
-terminalOutput (MkTerminal res _ _ _) = readIORef res
+terminalOutput (MkTerminal res _ _ _) = readIORef res >>= return . ($ "")
```

Comment:
why not `fmap ($ "") (readIORef res)` ?

Reply:
Either way works for me, the only thing I care about is to avoid `<$>` for backwards compatibility.